yarn.lock

```diff
@@ -2632,12 +2632,18 @@ __metadata:
   dependencies:
     "@lowdefy/api": 4.0.0-alpha.6
     "@lowdefy/block-utils": 4.0.0-alpha.6
+    "@lowdefy/blocks-antd": 4.0.0-alpha.6
```

Comment:
Lowdefy server should not have dep on lowdefy plugins

Reply:
Ahh. sorry this was part of another change.